scripts/check-site.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright 2019 Google LLC
```

Comment:
We should use 2021 when creating a new file.

Reply:
Done.